dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/IEC/BaseControls/WPP/LinearCommunicationWPRef.mo

```diff
@@ -0,0 +1,70 @@
+within Dynawo.Electrical.Controls.IEC.BaseControls.WPP;
```

Comment:
It looks like this model is exactly the same as LinearCommunicationPD, can't we just keep only one of them ?